lightning/src/ln/channel.rs

```diff
@@ -301,6 +301,21 @@ pub struct CounterpartyForwardingInfo {
 	pub cltv_expiry_delta: u16,
 }
 
+/// When a node increases the feerate on an outbound channel, it can cause the channel to get
+/// "stuck" if the other side holds most of the funds, and suddenly is unable to send an HTLC as it
+/// would effectively debit the channel initiator to pay for the new HTLC's fee. If the initiator's
+/// balance is right at or near the channel reserve, neither side will be able to send an HTLC.
+/// Thus, before sending an HTLC when we are the initiator, we check that the feerate can increase
+/// by this multiple without hitting this case, before sending.
+/// This multiple is effectively the maximum feerate "jump" we expect until more HTLCs flow over
+/// the channel. Sadly, there isn't really a good number for this - if we expect to have no new
+/// HTLCs for days we may need this to suffice for feerate increases across days, but that may
+/// leave the channel less usable as we hold a bigger reserve.
+#[cfg(fuzzing)]
+pub const CHAN_STUCK_FEE_INCREASE_MULTIPLE: u64 = 2;
+#[cfg(not(fuzzing))]
+const CHAN_STUCK_FEE_INCREASE_MULTIPLE: u64 = 2;
```

Comment:
I think the value is already mentioned as the fee spike buffer in few places, maybe `FEE_SPIKE_BUFFER_INCREASE_MULTIPLE` for consistency?